frontend/src/components/pages/ImportStoryPage.tsx

```diff
@@ -25,13 +27,13 @@ const ImportStoryPage = () => {
       <Flex
         direction="column"
         flex="1"
-        padding="40px 120px 40px 120px"
+        padding="40px 120px 60px 120px"
         overflow="auto"
       >
         <Link
           color="blue.100"
           fontWeight="bold"
-          href="#/"
+          href="#/?tab=1"
```

Comment:
Should the user be redirected to `tab=1` even if they accessed the import story page via `tab=0`?

Reply:
yeah, I think that makes sense since importing stories is more related to stories than story translations